Adapter.C

```diff
@@ -254,7 +254,14 @@ void preciceAdapter::Adapter::configure()
         DEBUG(adapterInfo("Creating interfaces..."));
         for (uint i = 0; i < interfacesConfig_.size(); i++)
         {
-            Interface* interface = new Interface(*precice_, mesh_, interfacesConfig_.at(i).meshName, interfacesConfig_.at(i).locationsType, interfacesConfig_.at(i).patchNames, interfacesConfig_.at(i).meshConnectivity, interfacesConfig_.at(i).resetDisplacement);
+            std::string namePointDisplacement = "pointDisplacement";
+            std::string nameCellDisplacement = "cellDisplacement";
```

Comment:
This is something we should either add in the adapter config or drop / hard-code. It does not really help to have the names here.

Reply:
I removed the hard-coded names.